### PR TITLE
[CI] Dependabot to run on all packages. Pin Packages. Remove unused packages

### LINKED ISSUE #4794: [CI] Remove dependabot whitelist
#### Summary
We've spent the last 4 months using a smaller whitelist of dependencies for dependabot. It's been working flawlessly for about 3 of those months. We should rip off the bandaid and let it manage the dependencies for us.

### PR DESCRIPTION
**Pull request by unlikelyzero**
Closes 
#4794 
#3178 
#4808

### Describe your changes:
This will remove the allowlist for dependabot and allow it to scan all of our dependencies for changes.
Pin as many packages as we can
Remove as many packages as we can

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by shefalijoshi** [comment]
We should disallow certain dependency updates like angular, webpack etc.

File: package.json [hunk 1]
```diff
@@ -64,23 +60,21 @@
...
-    "sass-loader": "^12.1.0",
-    "sinon": "^12.0.1",
-    "split": "^1.0.0",
+    "resolve-url-loader": "4.0.0",
+    "sass": "1.49.0",
+    "sass-loader": "12.4.0",
+    "sinon": "12.0.1",
     "style-loader": "^1.0.1",
     "uuid": "^3.3.3",
-    "v8-compile-cache": "^1.1.0",
```
Comment (unlikelyzero) [inline package.json:74]: Don't see any reference to this

File: package.json [hunk 2]
```diff
@@ -3,38 +3,37 @@
...
+    "eslint-plugin-vue": "7.5.0",
+    "eslint-plugin-you-dont-need-lodash-underscore": "6.10.0",
+    "eventemitter3": "1.2.0",
+    "exports-loader": "0.7.0",
+    "express": "4.13.1",
+    "file-loader": "6.1.0",
+    "file-saver": "1.3.8",
+    "git-rev-sync": "1.4.0",
+    "html-loader": "0.5.5",
+    "html2canvas": "1.4.1",
```
Comment (unlikelyzero) [inline package.json:33]: bump

File: package.json [hunk 3]
```diff
@@ -3,38 +3,37 @@
...
     "angular-route": "1.4.14",
     "babel-eslint": "10.1.0",
-    "comma-separated-values": "^3.6.4",
-    "concurrently": "^3.6.1",
-    "copy-webpack-plugin": "^10.2.0",
-    "cross-env": "^6.0.3",
-    "css-loader": "^4.0.0",
+    "comma-separated-values": "3.6.4",
+    "copy-webpack-plugin": "10.2.0",
+    "core-js": "3.20.3",
```
Comment (unlikelyzero) [inline package.json:16]: This is not a direct dependency but one which is used in many of the webpack / linting libraries

**Comment by unlikelyzero** [comment]
> We should disallow certain dependency updates like angular, webpack etc.

Since angular is going to be removed, I think we can use the @dependabot ignore syntax when that package comes up.

Webpack should be bumped now that we have some level of e2e coverage and an exhaustive node testing chain

**Comment by akhenry** [comment]
Agree, the dependabot workflow works better for our purposes.

### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A - build config only
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Review by akhenry — APPROVED** [review]

**Comment by khalidadil** [comment]
Looks good in Testathon on 03/17/2022

**Pull request merged**